src/Backend/Modules/FormBuilder/Ajax/SaveField.php

```diff
@@ -190,17 +190,17 @@ public function execute()
                             foreach ($postedValues as $postedValue) {
                                 $values[] = array(
                                     'value' => CommonUri::getUrl($postedValue),
-                                    'label' => $postedValue
+                                    'label' => $postedValue,
                                 );
                             }
                             if ($defaultValues != '') {
                                 $defaultValues = CommonUri::getUrl($defaultValues);
                             }
                         }
 
-                        /**
-                         * Save!
-                         */
+/**
+ * Save!
+ */
```

Comment:
it was because the multi line was formatted as a docblock, and the tool I used tries to match that with the function/class, that was not possible here, fixed in e14891b